data/transactions/logic/assembler_test.go

```diff
@@ -1302,13 +1302,13 @@ func TestAssembleDisassembleCycle(t *testing.T) {
 	// catch any suprises.
 	for v, source := range tests {
 		t.Run(fmt.Sprintf("v=%d", v), func(t *testing.T) {
-			ops := testProg(t, source, v)
+			ops := testProg(t, obfuscate(source), v)
```

Comment:
Why were you able to remove these?  I would expect you'd find whatever errors these were put in to suppress.

Reply:
I'm confused what you mean here, but the sources there all originally had branches, so they essentially had implicit obfuscates. I just had to add them explicitly b/c we no longer stop erroring on branches. You could argue that before it at least typechecked the part before the branch, but if you look at v1nonsense there are no type errors before the branches.